example/src/index.ts

```diff
@@ -74,8 +74,8 @@ const config: ResolveConfigFn = (env: Env, _trigger) => {
 			url: 'https://otel.baselime.io/v1',
 			headers: { 'x-api-key': env.BASELIME_KEY },
 		},
-		service: { name: 'greetings' },
```

Comment:
Might also consider setting namespace here - I just wasn't sure which namespace to set